### PR TITLE
Cover Brokers page with tests #213

### PR DESCRIPTION
**Pull request by MBovtriukProvectus**
Cover Brokers page with tests #213

### CODE REVIEW

File: kafka-ui-react-app/src/components/Brokers/__test__/BrokersContainer.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  const store = configureStore();
+  it('renders view', () => {
+    const component = shallow(
+      <Provider store={store}>
+        <BrokersContainer />
+      </Provider>
+    );
+
+    expect(component.exists()).toBeTruthy();
```
Comment (GneyHabub) [inline kafka-ui-react-app/src/components/Brokers/__test__/BrokersContainer.spec.tsx:20]: If you take a look at the implementation of `containerRendersView`, you'll see that it does almost exactly what you've written here. So this code might be redundant

Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Brokers/__test__/BrokersContainer.spec.tsx]: deleted

File: kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  it('matches snapshot', () => {
+    expect(mount(setupEmptyComponent())).toMatchSnapshot();
+  });
+
+  it('renders Brokers', () => {
+    const component = shallow(setupEmptyComponent());
+    expect(component.find('.section')).toBeTruthy();
+  });
+});
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx:45]: I want to see that component reacts correctly on isFetched change. Also would be great to add non empty component case

Comment (MBovtriukProvectus) [reply kafka-ui-react-app/src/components/Brokers/__test__/Brokers.spec.tsx:88]: done, if I correctly understand your comment

**Pull request merged**